src/main/java/io/lettuce/core/resource/EpollProvider.java

```diff
@@ -153,6 +156,13 @@ public SocketAddress newSocketAddress(String socketPath) {
             return null;
         }
 
+        @Override
+        public Class<? extends DatagramChannel> datagramChannelClass() {
```

Comment:
[Question; nits]
Should I add `@since` annotation here? Is this method regarded as `New public API`?
https://github.com/lettuce-io/lettuce-core/blob/main/.github/CONTRIBUTING.md#conventions